src/app/TableMsg.elm

```diff
@@ -0,0 +1,13 @@
+module TableMsg exposing (Msg(..))
```

Comment:
This should really be `Table.Msg`, however, because it is needed in `Types`, it cannot be, otherwise there's a circular reference.

I think that the `Types` module should be a simple module that does not depend on other modules. And other modules can use it to share common types. Making `Types` a smart module has gotten me into alot of trouble before, uneasy.